packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -190,40 +191,50 @@ export class TablePageObject<T extends TableRecord> {
     public getRenderedIconColumnCellIcon(
```

Comment:
For `getRenderedIconColumnCellIcon` and `getRenderedIconColumnGroupHeaderIcon` can we return a tagName string instead of an element instance? We have been trying to avoid page objects exposing element references.

I'm betting it'll improve the expect statements as well instead of `expect(blah instanceof Icon).toBeTrue()` we can expect a tag `expect(tagName).toBe(iconTagName)`

Reply:
The drawback would be that you wouldn't be able to get `Icon.severity`. Would you prefer we add a separate function for that?

Reply:
Yep!

Reply:
Okay, added two new functions: one to get the icon severity and one to get the Aria label.